definitions/npm/lodash_v4.x.x/flow_v0.63.x/lodash_v4.x.x.js

```diff
@@ -1608,7 +1607,7 @@ declare module "lodash/fp" {
 
   declare type NestedArray<T> = Array<Array<T>>;
 
-  declare type matchesIterateeShorthand = Object;
+  declare type matchesIterateeShorthand = {[string | number]: any};
```

Comment:
Now that `Object` means `any` an indexed object type more accurately describes that the `matchesIterateeShorthand` type can be.